x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/SpanFlyout/StickySpanProperties.tsx

```diff
@@ -49,7 +67,10 @@ export function StickySpanProperties({ span, totalDuration }: Props) {
 
   const spanName = span.span.name;
   const spanDuration = span.span.duration.us;
-  const spanTypeLabel = getSpanLabel(getPrimaryType(span.span.type));
+  const { type, subtype, action } = getSpanTypes(span);
+  const spanTypeLabel = getSpanLabel(type);
+  const spanSubtypeLabel = getSpanLabel(subtype);
+  const spanActionLabel = getSpanLabel(action);
```

Comment:
Would it make sense to split `getSpanLabel` into `getSpanTypeLabel`, `getSpanSubTypeLabel` and `getSpanActionLabel` ?

Reply:
... and can we drop the "Label" suffix? Makes me think it's the literal "Type", "Subtype" labels - but it's actually the values